classes/class-wc-rest-connect-shipping-label-controller.php

```diff
@@ -67,7 +64,7 @@ public function run( $request ) {
 				'tracking' => $label_data->label->tracking_id,
 				'refundable_amount' => $label_data->label->refundable_amount,
 				'created' => $label_data->label->created,
-				'carrier_id' => $settings[ 'carrier' ],
+				'carrier_id' => $label_data->label->carrier_id,
```

Comment:
@DanReyLop @marcinbot I'm seeing a case difference (all caps vs. lowercase) with this change that's causing the tracking link to not work before a page reload.